config/mail.php

```diff
@@ -44,6 +44,7 @@
             'password' => env('MAIL_PASSWORD'),
             'timeout' => null,
             'local_domain' => env('MAIL_EHLO_DOMAIN'),
+            'verify_peer' => env('MAIL_VERIFY_SSL', 'true'),
```

Comment:
Instead of passing `'true'` as a string can you update it to a boolean and pass `true` as the default?

Reply:
Small change, once updated i'll merge. Thanks!

Reply:
Will do.

Reply:
The new default value is commited.